portality/static/js/edges/admin.applications_and_update_requests.edge.js

```diff
@@ -59,7 +59,7 @@ $.extend(true, doaj, {
                     id: "application_status",
                     category: "facet",
                     field: "admin.application_status.exact",
-                    display: "Application Status",
+                    display: "Status",
```

Comment:
I'm attaching this comment here, but it's more broadly to do with this new file.  What I meant was for us to produce 2 new edges files, something like `admin.new_applications.edge.js` and `admin.update_requests.edge.js` and for the common elements of them to be factored into `doaj.fieldrender.edges.js` (like, facet definitions, result display definitions, etc).  Your approach here works fine until we want to vary the behaviours of the pages independently of each other, at which point we have to do that refactoring anyway.  So, I could be convinced to leave it as it is for the moment, but maybe we should check that the team don't want to change the list of facets for each page at this point, and if they do then we should go the extra distance to refactor.

Reply:
My intention was that the repeated bits of code would go into `doaj.fieldrender.edges.js` and be re-used by each of these components.  You can look at how the `inDOAJ` facet is handled in journals, for example

Reply:
See: https://github.com/DOAJ/doaj/blob/develop/portality/static/js/doaj.fieldrender.edges.js#L3 and https://github.com/DOAJ/doaj/blob/develop/portality/static/js/edges/admin.journals.edge.js#L78

So each interface is composed from a variety of standard components (ideally)